apps/web/src/pages/blob/[hash].tsx

```diff
@@ -87,9 +88,36 @@ const Blob: NextPage = function () {
       });
     }
     detailsFields.push(
-      { name: "Versioned Hash", value: blob.versionedHash },
-      { name: "Commitment", value: blob.commitment },
-      { name: "Proof", value: blob.proof }
+      {
+        name: "Versioned Hash",
+        value: (
+          <div className="flex items-center gap-2">
+            {blob.versionedHash}
+            <CopyToClipboard
+              value={blob.versionedHash}
+              label="Copy versioned hash"
+            />
+          </div>
```

Comment:
I’m thinking abstracting this into a new component that takes in a `label` and `value`, and then renders a `div` containing the value along with the `CopyToClipboard` component. What do you think?

We could name it something like `Copyable`.

wdyt?

Reply:
Done [386cd1a](https://github.com/Blobscan/blobscan/pull/478/commits/386cd1aa22f7d42021daf8de6bb6a77880df3813)